src/schema/v2/sale/index.ts

```diff
@@ -9,7 +9,7 @@ import cached from "schema/v2/fields/cached"
 import date from "schema/v2/fields/date"
 import moment from "moment"
 import { SlugAndInternalIDFields } from "schema/v2/object_identification"
-import { formattedStartDateTime } from "lib/date"
+import { DEFAULT_TZ, formattedStartDateTime } from "lib/date"
```

Comment:
It sort of seems odd that types are importing this for use in resolvers. It seems like the date format/helpers being used should make the determination (of falling back to `DEFAULT_TZ` where needed), vs. clients here passing it in.

#minor #unrelated